cmd/quota.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func checkQuota(container Container, result *Result) {
+	if container.Resources.Limits == nil {
+		occ := Occurrence{id: ErrorResourcesLimitNIL, kind: Error, message: "Quota not set, please set it!"}
+		result.Occurrences = append(result.Occurrences, occ)
+		return
+	}
+
+	if container.Resources.Limits.Cpu() == nil || container.Resources.Limits.Cpu().IsZero() {
+		occ := Occurrence{id: ErrorResourcesLimitNIL, kind: Error, message: "CPU quota not set, please set it!"}
+		result.Occurrences = append(result.Occurrences, occ)
+	}
+
+	if container.Resources.Limits.Memory() == nil || container.Resources.Limits.Memory().IsZero() {
+		occ := Occurrence{id: ErrorResourcesLimitNIL, kind: Error, message: "Memory quota not set, please set it!"}
+		result.Occurrences = append(result.Occurrences, occ)
+	}
+
+	if len(result.Occurrences) == 0 {
```

Comment:
I am not sure we want to log an info message if everything is configured correctly because we don't do that for any other command. The only time we log is for an image audit and there only because it makes sense to tell that the matching image was found.

Reply:
I add to test when cpu and memory limits are ok (like image command). But I prefer to find a better way. I remove it.